q2_feature_table/tests/filter/test_conditional_filter.py

```diff
@@ -40,6 +40,48 @@ def test_conditional(self):
         npt.assert_array_equal(known.ids(axis='observation'),
                                test_.ids(axis='observation'))
 
+    def test_allow_empty_table_true(self):
+        # test True
```

Comment:
I'm not seeing anywhere in this test where we confirm that this table is in fact empty - you have the `allow_empty_table` set to True, but it would be good to add an assertion with an 'expected' table that's empty that you're asserting is equal to the actual table that results from this method.